projects/erp-hgrid/src/services/custom-operations/custom-sorting-strategy.ts

```diff
@@ -0,0 +1,20 @@
+export class AddressSortStrategy {
+    constructor(private formatter: (value: any) => string) {}
+  
+    compareValues(a: any, b: any): number {
+      const addressA = this.formatter(a)?.toLowerCase() ?? '';
+      const addressB = this.formatter(b)?.toLowerCase() ?? '';
+      return addressA.localeCompare(addressB);
```

Comment:
Not sure if `localeCompare` is the best way to sort addresses.

There could be some unexpected results:
![image](https://github.com/user-attachments/assets/5371d7b1-239d-48e8-b1aa-51aa0764c6f4)

Starting to think if we should simply disable the sorting for adress and full address. How about that?